test/parallel/test-fs-readfile-error.js

```diff
@@ -46,18 +46,14 @@ function test(env, cb) {
   }));
 }
 
-test({ NODE_DEBUG: '' }, common.mustCall((data) => {
-  assert(/EISDIR/.test(data));
-  assert(!/test-fs-readfile-error/.test(data));
-}));
```

Comment:
As question: what was the reason that this was removed?

Reply:
The test won't pass because of:

```js
  Error.captureStackTrace(err, validatePath);
```